src/App.js

```diff
@@ -1,46 +1,33 @@
 import './App.css';
 import {
-    Title,
-    Counter,
-    Container,
-    InputField,
-    GenreSelect,
-    MovieContainer,
     MovieDetails,
-    SortControl,
-    MovieForm
+    MovieListPage,
+    SearchForm
 } from "./components";
-import {moviesArray} from "./mockedMovies";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+export const routes = createBrowserRouter([
```

Comment:
Please, create new file for routes. And btw, why don't you use classic approach with <Routers>, I do not see any SSR in your task?

Reply:
Created separate file.
I used the approach from documentation